src/KubernetesClient/KubernetesClientConfiguration.ConfigFile.cs

```diff
@@ -689,6 +689,23 @@ public static K8SConfiguration LoadKubeConfig(Stream kubeconfigStream)
             return LoadKubeConfigAsync(kubeconfigStream).GetAwaiter().GetResult();
         }
 
+        /// <summary>
+        ///     Saves Kube Config
+        /// </summary>
+        /// <param name="kubeconfig">Instance of the <see cref="K8SConfiguration"/> class to save</param>
+        /// <param name="kubeconfigPath">Path to save the kubeconfig file or <see cref="KubeConfigDefaultLocation"/> if not provided</param>
+        public static void SaveKubeConfig(K8SConfiguration kubeconfig, string kubeconfigPath = null)
```

Comment:
Suggest just `Save(...)` since `KubernetesClientConfiguration.ConfigFile.SaveKubeConfig(...)` is kind of redundant.